src/Hedgehog/Extras/Test/Golden.hs

```diff
@@ -53,6 +68,9 @@ diffVsGoldenFile
   -> FilePath -- ^ Reference file
   -> m ()
 diffVsGoldenFile actualContent referenceFile = GHC.withFrozenCallStack $ do
+  forM_ mGoldenFileLogFile $ \logFile ->
```

Comment:
It would be nice to mention the new environment variable in the comment.

Reply:
Done!